packages/sftp/test/index.js

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { execute } from '../src';
 
 describe('The execute() function', () => {
-  it('executes each operation in sequence', done => {
+  it.skip('executes each operation in sequence', done => {
```

Comment:
@josephjclark it's because we are establishing a connection to sftp client on `execute`. So that means we need a live sftp server for the execute test to work 
```
return state =>
    commonExecute(
      connect,
      ...operations,
      disconnect
    )({ ...initialState, ...state });
    ```

Reply:
Ah I see - so we need to work out a mock. Ok no problem - thanks!

Reply:
@josephjclark and @mtuchi , FWIW i've got that pattern laid down [here](https://github.com/OpenFn/adaptors/blob/main/packages/template/test/mockAgent.js) but it's not been implemented in many adaptors yet.